worker/runtime/resolvconf_parser.go

```diff
@@ -0,0 +1,54 @@
+package runtime
+
+import (
+	"fmt"
+	"io/ioutil"
+	"regexp"
+	"strings"
+
+	"code.cloudfoundry.org/localip"
+)
+
+// Parse resolve.conf file from the provided path.
```

Comment:
Might also be worth referencing [resolv_compiler](https://github.com/cloudfoundry/guardian/blob/5ccf9693d394774f17a46de240f31b90597bd70f/kawasaki/dns/resolv_compiler.go) so folks can dig into where some of the reasoning comes from.